hpOneView/resources/security/login_details.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright (2012-2017) Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+###
+
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+
+standard_library.install_aliases()
+
+
+from hpOneView.resources.resource import ResourceClient
+from hpOneView.exceptions import HPOneViewException
+
+
+class Login_Details(object):
+    """
+    list login details.
+
+    """
+
+    URI = '/rest/logindetails'
+
+    DEFAULT_VALUES = {
+        '200': {'type': 'logindetails'},
+        '300': {'type': 'logindetails'},
+        '500': {'type': 'logindetails'}
+    }
+
+    def __init__(self, con):
+        self._connection = con
+        self._client = ResourceClient(con, self.URI)
+
+    def get_all(self):
```

Comment:
Not sure if `get_all` would be the name for the method we'd want to go with, since it will not be a list containing several individual 'login details', nor will it be filtered or anything like a usual `get_all`

Perhaps `get_login_details`? Or even just `get`, but `get_all` does seem a bit off.